spec/client_spec.rb

```diff
@@ -62,8 +62,11 @@ def send_stream_content(res, content, keep_open:)
         expect(received_req.header).to eq({
           "accept" => ["text/event-stream"],
           "cache-control" => ["no-cache"],
-          "host" => ["127.0.0.1"],
-          "authorization" => ["secret"]
+          "host" => ["127.0.0.1:" + server.port.to_s],
+          "authorization" => ["secret"],
+          "user-agent" => ["ruby-eventsource"],
+          "content-length" => ["0"],
```

Comment:
Is this just an unavoidable behavior of httprb - sending Content-Length: 0 for a GET request? Seems a bit nonstandard per [RFC7230](https://tools.ietf.org/html/rfc7230#section-3.3.2), as Get isn't supposed to have a body ("A user agent SHOULD NOT send a Content-Length header field when the request message does not contain a payload body and the method semantics do not anticipate such a body"), but I guess it doesn't do any harm.

Reply:
I could not find a way to not do that through the public API after reading the source of the library some more. I Read through the changelog and someone had an issue with `POST` requests with an empty body and their PR added the length for everything. I don't think this hurts anything.